les/checkpointoracle/oracle.go

```diff
@@ -88,6 +95,12 @@ func (oracle *CheckpointOracle) Contract() *checkpointoracle.CheckpointOracle {
 // StableCheckpoint returns the stable checkpoint which was generated by local
 // indexers and announced by trusted signers.
 func (oracle *CheckpointOracle) StableCheckpoint() (*params.TrustedCheckpoint, uint64) {
+	oracle.checkMu.Lock()
+	defer oracle.checkMu.Unlock()
+	if time.Since(oracle.lastCheckTime) < 1*time.Second {
```

Comment:
Perhaps we can extend this threshold to 1 minute. 1 minute is around 4 blocks.

It's totally fine even in these 4 blocks the registered checkpoint has been updated, the downside is the light client may use the legacy checkpoint instead of the latest one.